### ISSUE TITLE
Crash on startup with react-native 0.33.0 on iOS7

### ISSUE THREAD

**Issue by chetstone** [Microsoft/react-native-code-push#517]
Using react-native-code-push 1.14.6-beta, testing a release build on my iOS7 device, crashes with
`[NSOperationQueue setUnderlyingQueue:]: unrecognized selector sent to instance 0x14ee9a70` in
function
```
- (NSURLSessionDataTask *)sendRequest:(NSURLRequest *)request
                         withDelegate:(id<RCTURLRequestDelegate>)delegate
```
in `RCTHTTPRequestHandler.m`.

Works OK on iOS 9 device and iOS 8 simulator, also on iOS7 with debug build (no codepush). 
Everything worked on react-native 0.32.1.  I would like to use 0.33 which is the last version of react-native to support iOS7.

**Comment by silhouettes** [comment]
Thanks for filing this. RN 0.33 is not on our list of supported versions yet, but we will test it and iron out any issues over the next two days before updating the docs/pushing out a release. Thanks for your patience!

**Comment by legander** [comment]
Any news yet? @silhouettes

**Comment by silhouettes** [comment]
@chetstone - sorry about the delay. I will try to look at this today!

**Comment by silhouettes** [comment]
@chetstone, @legander - Sorry for the delay. I'm having significant difficulty getting my hands on an iOS7 device, or even simulator (it is not supported at all on OS X El Capitan).

Is iOS7 support truly important to you, keeping in mind that less than 3% of iOS devices are running it (and decreasing everyday)? Are you sure that it's a CodePush-specific problem?

I think that what you are looking for may be this PR submitted to React Native, specifically for networking calls when using RN 0.33 with iOS 7: https://github.com/facebook/react-native/pull/10013/files

**Comment by chetstone** [comment]
@silhouettes Thanks for looking into this. Yes it certainly looks like that's the issue. I've asked if they could merge the fix to 0.33.1 as I have several customers using iOS7 and there's one more feature in 0.33 I'd like to add before support goes away. It doesn't seem like it's a code-push problem though.
